packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js

```diff
@@ -282,6 +282,7 @@ const ReactNativeViewConfig = {
       flexGrow: true,
       flexShrink: true,
       flexWrap: true,
+      focusable: true,
```

Comment:
This doesn't seem quite right. focusable is a builtin prop. Do you know why we need to add it to the ViewConfig and why for style?

Reply:
Removed.